plexus-compilers/plexus-compiler-javac-errorprone/src/test/java/org/codehaus/plexus/compiler/javac/JavacErrorProneCompilerTest.java

```diff
@@ -43,31 +40,69 @@
     extends AbstractCompilerTest
 {
 
+    protected boolean java8() {
+        return System.getProperty( "java.version" ).startsWith( "1.8" );
+    }
+    
+    private ClassLoader originalTCCL;
+    
     public void setUp()
         throws Exception
     {
         super.setUp();
-        setForceJavacCompilerUse( true );
+        if ( java8() ) {
```

Comment:
does it mean we do not test with new jdk such 14?

Reply:
I did test on LTS jdk8 and jdk11, and warning count changes ; I did not test with other jdk

Reply:
I did test on LTS jdk8 and jdk11, and warning count changes ; I did not test with other jdk

Reply:
I disable the `j2objc` test for ghactions.
Still some issues with jdk14

Reply:
@nicerloop rebase with master I have disabled jdk14 until we get this fixed

Reply:
I locally have issues with jdk14 in plexus-compiler-eclipse("The type java.lang.Object cannot be resolved") and plexus-compiler-javac (source/target 1.6 not supported anymore, use 7 or later)

Reply:
It seems to me master does not build with jdk14